liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/SliderDTO.kt

```diff
@@ -1,67 +1,205 @@
 package org.phoenixframework.liveview.data.dto
 
+import androidx.compose.foundation.interaction.MutableInteractionSource
 import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.RangeSlider
 import androidx.compose.material3.Slider
 import androidx.compose.material3.SliderColors
 import androidx.compose.material3.SliderDefaults
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableFloatStateOf
+import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.runtime.snapshotFlow
 import androidx.compose.ui.graphics.Color
 import kotlinx.collections.immutable.ImmutableMap
 import kotlinx.collections.immutable.toImmutableMap
 import org.phoenixframework.liveview.data.core.CoreAttribute
+import org.phoenixframework.liveview.data.dto.SliderDtoFactory.endThumb
+import org.phoenixframework.liveview.data.dto.SliderDtoFactory.startThumb
+import org.phoenixframework.liveview.data.dto.SliderDtoFactory.thumb
+import org.phoenixframework.liveview.data.dto.SliderDtoFactory.track
 import org.phoenixframework.liveview.data.mappers.JsonParser
+import org.phoenixframework.liveview.domain.base.ComposableBuilder
+import org.phoenixframework.liveview.domain.base.ComposableTypes
 import org.phoenixframework.liveview.domain.base.ComposableViewFactory
 import org.phoenixframework.liveview.domain.base.PushEvent
 import org.phoenixframework.liveview.domain.extensions.privateField
 import org.phoenixframework.liveview.domain.extensions.toColor
 import org.phoenixframework.liveview.domain.factory.ComposableTreeNode
+import org.phoenixframework.liveview.ui.phx_components.PhxLiveView
 
 /**
  * Material Design slider.
  * ```
- * <Slider value={"#{@sliderValue}"} phx-change="setSliderValue" minValue="0" maxValue="100" />
+ * // sliderValue is a float
+ * <Slider
+ *   value={"#{@sliderValue}"}
+ *   phx-change="setSliderValue"
+ *   minValue="0"
+ *   maxValue="100" />
+ * // sliderRange is an array of two positions
+ * <RangeSlider
+ *   value={"#{Enum.join(@sliderRange, ",")}"}
```

Comment:
I think `Enum.join` will produce a string, so you may not need the string interpolation as well.